components/ModEditHealthcareProfessionalSection.vue

```diff
@@ -0,0 +1,279 @@
+                    {{ degree }}
+                </option>
+            </select>
+            <label
+                for="Specialties"
+                class="my-2 text-primary-text text-sm font-bold font-sans"
+            >
+                {{ $t("modHealthcareProfessionalSection.selectSpecialties") }}
+            </label>
+            <select
+                id="healthcare-professional-specialties"
+                v-model="sectionFields.healthcareProfessionalSpecialties"
+                data-testid="mod-healthcare-professional-section-specialties"
+                name="Specialties"
+                multiple
+                class="mb-5 px-3 py-3.5 w-96 h-32 bg-secondary-bg rounded-lg border border-primary-text-muted
+                            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+            >
+                <option
+                    v-for="(specialty, index) in specialtyOptions"
+                    :key="`${specialty}-${index}`"
+                    :value="specialty"
+                >
+                    {{ specialty }}
+                </option>
+            </select>
+            <label
+                for="Locales"
+                class="my-2 text-primary-text text-sm font-bold font-sans"
+            >
+                {{ $t("modHealthcareProfessionalSection.selectLocales") }}
+            </label>
+            <select
+                id="healthcare-professional-locales"
+                v-model="sectionFields.healthcareProfessionalLocales"
+                data-testid="mod-healthcare-professional-section-spoken-locales"
+                name="Locales"
+                multiple
+                class="mb-5 px-3 py-3.5 w-96 h-32 bg-secondary-bg rounded-lg border border-primary-text-muted
+                            text-primary-text text-sm font-normal font-sans placeholder-primary-text-muted"
+            >
+                <option
+                    v-for="(locale, index) in localeOptions"
+                    :key="`${locale}-${index}`"
+                    :value="locale"
+                >
+                    {{ locale }}
+                </option>
+            </select>
+        </div>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { onMounted } from 'vue'
+import { ModerationScreen, useModerationScreenStore } from '~/stores/moderationScreenStore'
+import { useHealthcareProfessionalsStore } from '~/stores/healthcareProfessionalsStore'
+import { Locale, type LocalizedNameInput, Insurance, Degree, Specialty } from '~/typedefs/gqlTypes'
+import { multiSelectWithoutKeyboard } from '~/utils/multiSelectWithoutKeyboard'
+import SVGTrashCan from '~/assets/icons/trash-can.svg'
+import SVGProfileIcon from '~/assets/icons/profile-icon.svg'
+
+const moderationScreenStore = useModerationScreenStore()
+const healthcareProfessionalsStore = useHealthcareProfessionalsStore()
+// less clear name necessary because of filter rules on line length (max = 130)
```

Comment:
A lot of comments are being added with this change. We should try to avoid adding comments to the component files whenever possible. Instead of adding additional comments, try to refactor your code in a way that it's easy to understand without the comment.

I'm not 100% sure what is meant by this one, but I'm guessing line 220 was too long? You could easily fix that by moving it one line down and replacing `healthcareProfessionalsStore.healthcareProfessionalSectionFields` with the nifty `sectionFields` variable you have.

Example:
```
const sectionFields = healthcareProfessionalsStore.healthcareProfessionalSectionFields
const healthcareProfessionalNameArray = sectionFields.healthcareProfessionalNameArray

```